### ISSUE TITLE
Hyperspin artwork wrong orientation

### ISSUE THREAD

**Issue by nikosrozos** [mickelson/attract#623]
latest attract-mode 2.6.1 loads some hyperspin themes upside down.
I have also installed 2.3.0 version which seems to load them correctly.
some examples are rtypeleo, radm, prehisle to name a few.

attract -v
Attract-Mode v2.6.1 (Linux, SFML 2.4.2 +FontConfig +Xinerama +SWF +7z) 
avcodec 57.107.100 / avformat 57.83.100 / swscale 4.8.100 / avutil 55.78.100 / swresample 2.9.100

**Comment by MancV21** [comment]
Yeah, same issue here. The "Next Game" part of the theme is upside down. Everything else (videos and other artwork) are fine.

**Comment by pcca-matrix** [comment]
there are a very large number of bugs with flash medias on AM , the reverse is one of them , i was able to bypass the swf reverse bug with some glsl code.

You can try my attract-mode layout 100% compatible with hyperspin install, it is intended to run on windows but it would be good to test it under linux.

https://github.com/pcca-matrix/PCCA-Layout

**Comment by mickelson** [comment]
hmm I wonder what changed bewteen 2.3 and 2.6... can you print the attract -v output on yoru 2.3 version that works (so I can see the sfml version) thanks

**Comment by nikosrozos** [comment]
./attract -v
Attract-Mode v2.3.0 (Linux, SFML 2.4 +FontConfig +Xinerama +SWF +7z) 
avcodec 57.107.100 / avformat 57.83.100 / swscale 4.8.100 / swresample 2.9.100

Shaders are available.

--------------------------------------------------------------------

./attract -v
Attract-Mode v2.6.1 (Linux, SFML 2.4.2 +FontConfig +Xinerama +SWF +7z) 
avcodec 58.35.100 / avformat 58.20.100 / swscale 5.3.100 / avutil 56.22.100 / swresample 3.3.100

OpenGL 4.6.0 NVIDIA 440.82
 - vendor  : NVIDIA Corporation
 - renderer: GeForce GTX 960/PCIe/SSE2

Shaders are available.

**Comment by sergiohf** [comment]
I made a clean installation of ubuntu 18.04 minimal in my asus with intel hd 4000 graphics. When i compiled attractmode, as described in wiki the hyperspin.nut works ok, but there no vaapi driver, so the pcca layout didnt work. So i recompiled attractmode, modifying makefile and enabling vaapi. After that the layout and animations worked, but the swf files are upside down and mirrored. Look the pictures and the log file.
![IMG_20200705_031804](https://user-images.githubusercontent.com/59039520/86542541-deef3780-beec-11ea-9b11-72eddef15f73.jpg)
![IMG_20200705_031732](https://user-images.githubusercontent.com/59039520/86542543-e7477280-beec-11ea-9eee-355941fbbe50.jpg)
[log.txt](https://github.com/mickelson/attract/files/4875690/log.txt)

**Comment by mickelson** [comment]
seems to be this commit that messed it up: https://github.com/mickelson/attract/commit/553cc8e4d4120b08d18c95c1dd16137917578953#diff-05d50e35b8a15804bfa87b11865eb0b9

**Comment by nikosrozos** [comment]
Well, I recently installed ubuntu 20.04, compiled attract mode and
everything was fine. No more upside down apart from some misallignments.

Στις Δευ, 6 Ιουλ 2020, 03:39 ο χρήστης Andrew Mickelson <
notifications@github.com> έγραψε:

**Comment by sergiohf** [comment]
You are right, i updated to ubuntu 20.04 and it works

**Comment by Metalciaf** [comment]
got the same problem :(
but only with .swf  image 

raspberry pi4 
retropie 4.8

Attract-Mode v2.6.1 (Linux, SFML 2.4.2 +FontConfig +SWF +Curl)
avcodec 58.90.100 / avformat 58.44.100 / swscale 5.6.101 / avutil 56.49.100 / swresample 3.6.100

OpenGL 2.1 Mesa 19.3.2
 - vendor  : Broadcom
 - renderer: V3D 4.2

if with 2.3 work how can i rollback to the version? 

thank you
